error_test.go

```diff
@@ -66,7 +66,7 @@ func TestError(t *testing.T) {
 				result := NewError()
 
 				// ASSERT
-				test.That(t, result).Equals(&Error{statusCode: 500})
+				test.That(t, *result).Equals(Error{statusCode: 500})
```

Comment:
The remaining fields of the resulting Error{} are expected to contain their zero values; explicit initialisation of zero value fields in the wanted value in the test is unnecessary as Go ensures that fields are initialised with zero-values if not otherwise explicitly set.